modules/webaudio/AudioMixer.js

```diff
@@ -0,0 +1,94 @@
+/* global
+    __filename
+*/
+
+import { getLogger } from 'jitsi-meet-logger';
+import { createAudioContext } from './WebAudioUtils';
+
+const logger = getLogger(__filename);
+
+/**
+ * The AudioMixer, as the name implies, mixes a number of MediaStreams containing audio tracks into a single
+ * MediaStream.
+ */
+export default class AudioMixer {
+    /**
+     * Create AudioMixer instance.
+     */
+    constructor() {
+        this._started = false;
+        this._streamsToMix = [];
+    }
+
+    /**
+     * Adds the MediaStream from a JitsiLocalTrack to the list of streams to be mixed.
+     *
+     * @param {JitsiLocalTrack} jitsiLocalTrack - JitsiLocalTrack containing audio stream.
+     */
+    addJitsiLocalTrack(jitsiLocalTrack) {
```

Comment:
I mentioneed it in the other PR, but for completelenes:  I think we should drop this function and just deal with streams in this entity

Reply:
Please see my comment about this in the associated jitsi-meet PR.